addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -590,6 +590,12 @@
 			}
 		}
 		
+		"220"	//Shortstop
+		{
+			"desp"			"Shortstop: {positive}+35% faster reload time"
+			"attrib"		"97 ; 1.35"
```

Comment:
Meant to be `0.65` instead of `1.35`, this attrib makes smaller number quicker to reload while bigger number longer to reload